instance/models/openedx_appserver.py

```diff
@@ -222,6 +223,22 @@ def create_lms_user_settings(self):
         template = loader.get_template(self.LMS_USER_VARS_TEMPLATE)
         return template.render(dict(lms_users=self.lms_users.all()))
 
+    def smtp_relay_settings(self):
```

Comment:
@mtyaka I think we could make this a `@property` (just like `github_admin_username_list`).